src/lib/transport.js

```diff
@@ -37,17 +40,24 @@ function createPacketAndCondition ({
   const address = destinationAccount + '.~' + protocol + '.' + receiverId +
     (id ? ('.' + id) : '')
```

Comment:
Agreed. That's what I did as well. (When creating the example packet for the ilp-packet-viewer.)